zenith_utils/Cargo.toml

```diff
@@ -18,12 +18,9 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1"
 thiserror = "1.0"
 tokio = "1.11"
-
-slog-async = "2.6.0"
-slog-stdlog = "4.1.0"
-slog-scope = "4.4.0"
-slog-term = "2.8.0"
-slog = "2.7.0"
+tracing = "0.1.27"
+tracing-log = "0.1.2"
```

Comment:
See the first bullet in 
https://docs.rs/tracing-subscriber/0.2.24/tracing_subscriber/#optional-dependencies

tracing-subscriber with default features is enough to init tracing backend with `log` redirected inside it, see my branch for details.

Reply:
Good catch, I've missed that part too, because had called `.finish();` on my subscriber builder...
Like I've mentioned, the configuration is somewhat convoluted here.

Here's the difference in the in their sources: https://github.com/tokio-rs/tracing/blob/6b7051c9897a6ea2afd61cfa139714cfeed8bf4f/tracing-subscriber/src/fmt/mod.rs#L556-L579

the main bit is

```rust
/// If the `tracing-log` feature is enabled, this will also install
/// the LogTracer to convert `Log` records into `tracing` `Event`s.
```

that feature is enabled with the default features, so you should have it, then you should either invoke the right method on the formatter, or do this manually:
https://github.com/tokio-rs/tracing/blob/master/tracing-subscriber/src/util.rs#L57-L67